cg/cli/upload/genotype.py

```diff
@@ -25,9 +25,9 @@ def genotypes(context, re_upload, family_id):
     gt_api = context.obj["genotype_api"]
     hk_api = context.obj["housekeeper_api"]
     status_api = context.obj["status_db"]
-    family_obj = status_api.family(family_id)
+    case_obj = status_api.family(family_id)
```

Comment:
case_id?

Reply:
That would be nice but I think it would change the interface in a way that expands the scope too much for this PR